nodebuilder/tests/swamp/swamp_tx.go

```diff
@@ -12,7 +12,7 @@ func (s *Swamp) FillBlocks(ctx context.Context, bsize, blocks int) chan error {
 	errCh := make(chan error)
 	go func() {
 		// TODO: FillBlock must respect the context
-		_, err := testnode.FillBlock(s.ClientContext, bsize, s.accounts[:blocks+1])
+		_, err := s.ClientContext.FillBlock(bsize, s.accounts, flags.BroadcastBlock)
```

Comment:
Shouldn't this method still somehow continue to respect `blocks` param? 
```
produces the given amount of contiguous blocks
```

Reply:
oh shiieet, whoops good catch!! https://github.com/celestiaorg/celestia-node/pull/1147/commits/ce0f8341a8be2293f0bb6258973116d29f398798